components/ntp_background_images/browser/view_counter_service.cc

```diff
@@ -165,14 +183,23 @@ NTPBackgroundImagesData* ViewCounterService::GetCurrentWallpaperData() const {
 
 NTPSponsoredImagesData* ViewCounterService::GetCurrentBrandedWallpaperData()
     const {
-  NTPSponsoredImagesData* const images_data =
-      background_images_service_->GetBrandedImagesData(/*super_referral=*/true);
-  if (images_data && IsSuperReferralWallpaperOptedIn()) {
-    return images_data;
+  const ContentSetting content_setting =
+      host_content_settings_map_->GetContentSetting(
+          /*primary_url=*/GURL(), /*secondary_url=*/GURL(),
```

Comment:
does setting both of these URLs to empty just get the global setting?

Reply:
It does. Thanks